lib/populate_env/heroku/compilation.rb

```diff
@@ -31,7 +31,10 @@ def content
       end
       
       def perform
+        old_umask = File.umask
+        File.umask(077)
```

Comment:
@malclocke Good idea! Can we turn this into a constant? I'm not sure what `077` means.